### ISSUE TITLE
mac launch failure

### ISSUE THREAD

**Issue by Callendor** [NvChad/NvChad#525]
**Describe the bug**
After I install and launch nvim, I get only partially loaded packer plugins and I'm unable to get NvChad to execute successfully.  I have JetBrains Mono Nerd Font installed and I get a weirdly formatted screen that looks like this:

<img width="1525" alt="Screen Shot 2021-10-02 at 11 47 29 PM" src="https://user-images.githubusercontent.com/35014096/135741715-942116f7-7ea4-4464-8103-34c57bceab4c.png">

**To Reproduce**
1. install neovim with brew
2. follow NVCHad installation instructions
3. git clone https://github.com/NvChad/NvChad ~/.config/nvim
4. nvim +'hi NormalFloat guibg=#1e222a' +PackerSync
5. run nvim on existing file or run nvim by itself
6. When I run nvim by itself I don't get a dashboard screen but the same screen as posted above

**Expected behavior**
Expecting to see a dashboard when I run nvim by itself.  Expecting NvChad to work with mac os.

**Screenshots**

**Desktop (please complete the following information):**
 - Operating System -- macOS Big Sur Version 11.6
 - Terminal -- tried both iterm2 and alacritty
 - Version of Neovim -- NVIM v0.5.1 (Installed via Brew)

**Additional context**
I ran PackerStatus and saw this output:

<img width="1220" alt="Screen Shot 2021-10-02 at 11 51 49 PM" src="https://user-images.githubusercontent.com/35014096/135741834-88595f91-a9b5-4849-9b51-830de80e06e3.png">

Basically a ton of plugins that are listed as not loaded.  I've tried running PackerInstall but nothing executes...feels like maybe something is broken with Packer for mac users?

**Comment by siduck76** [comment]
get through the docs https://nvchad.github.io/config and you'll know how to enable the dashboard , set the nerdfont in your terminal too. Just installing it wont do.

**Comment by macarthy** [comment]
SAme OSX  and terminals here, all working,.  In Alacritty i have 

``` 
font:
  normal:
    family: 'SauceCodePro Nerd Font'

```

**Comment by ashincoder** [comment]
> **Describe the bug** After I install and launch nvim, I get only partially loaded packer plugins and I'm unable to get NvChad to execute successfully. I have JetBrains Mono Nerd Font installed and I get a weirdly formatted screen that looks like this:
> 
> <img alt="Screen Shot 2021-10-02 at 11 47 29 PM" width="1525" src="https://user-images.githubusercontent.com/35014096/135741715-942116f7-7ea4-4464-8103-34c57bceab4c.png">
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. install neovim with brew
> 2. follow NVCHad installation instructions
> 3. git clone https://github.com/NvChad/NvChad ~/.config/nvim
> 4. nvim +'hi NormalFloat guibg=#1e222a' +PackerSync
> 5. run nvim on existing file or run nvim by itself
> 6. When I run nvim by itself I don't get a dashboard screen but the same screen as posted above
> 
> **Expected behavior** Expecting to see a dashboard when I run nvim by itself. Expecting NvChad to work with mac os.
> 
> **Screenshots** If applicable, add screenshots to help explain your problem.
> 
> **Desktop (please complete the following information):**
> 
> * Operating System -- macOS Big Sur Version 11.6
> * Terminal -- tried both iterm2 and alacritty
> * Version of Neovim -- NVIM v0.5.1 (Installed via Brew)
> 
> **Additional context** I ran PackerStatus and saw this output:
> 
> <img alt="Screen Shot 2021-10-02 at 11 51 49 PM" width="1220" src="https://user-images.githubusercontent.com/35014096/135741834-88595f91-a9b5-4849-9b51-830de80e06e3.png">
> 
> Basically a ton of plugins that are listed as not loaded. I've tried running PackerInstall but nothing executes...feels like maybe something is broken with Packer for mac users?

@Callendor is the issue solved after setting nerd font in your terminal? 

If it isn't I recommend you to try kitty terminal https://github.com/kovidgoyal/kitty. Kitty terminal supports rendering icons and stuff.

**Comment by siduck76** [comment]
@ashincoder kitty does better font rendering by default unlike other terminals , in other terminals you have to set the nerd font manually in their configs but kitty does it by default

**Comment by ashincoder** [comment]
> @ashincoder kitty does better font rendering by default unlike other terminals , in other terminals you have to set the nerd font manually in their configs but kitty does it by default

@siduck76 That is what I said. Kitty has better font rendering support than other terminals. I just said alacritty to convince @Callendor.

**Comment by Callendor** [comment]
Switching to kitty fixed this. Sorry for the late reply.

From: Ashin Antony ***@***.***>
Reply-To: NvChad/NvChad ***@***.***>
Date: Sunday, October 3, 2021 at 10:48 PM
To: NvChad/NvChad ***@***.***>
Cc: Callendor ***@***.***>, Mention ***@***.***>
Subject: Re: [NvChad/NvChad] mac launch failure (#525)


Describe the bug After I install and launch nvim, I get only partially loaded packer plugins and I'm unable to get NvChad to execute successfully. I have JetBrains Mono Nerd Font installed and I get a weirdly formatted screen that looks like this:
[Image removed by sender. Screen Shot 2021-10-02 at 11 47 29 PM]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F35014096%2F135741715-942116f7-7ea4-4464-8103-34c57bceab4c.png&data=04%7C01%7C%7C6e23c675929d49a5411008d986f23f75%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637689197272890500%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=M%2B5vGTMe8AKh%2BLYtzPUNqTmqmAabCXzDF1LAyW0jlVI%3D&reserved=0>

To Reproduce Steps to reproduce the behavior:

  1.  install neovim with brew
  2.  follow NVCHad installation instructions
  3.  git clone https://github.com/NvChad/NvChad<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNvChad%2FNvChad&data=04%7C01%7C%7C6e23c675929d49a5411008d986f23f75%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637689197272900491%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=54iwgC2uIrHU57%2Bokw2vxqFECjLxXRkOquAvN4P34fU%3D&reserved=0> ~/.config/nvim
  4.  nvim +'hi NormalFloat guibg=#1e222a' +PackerSync
  5.  run nvim on existing file or run nvim by itself
  6.  When I run nvim by itself I don't get a dashboard screen but the same screen as posted above

Expected behavior Expecting to see a dashboard when I run nvim by itself. Expecting NvChad to work with mac os.

Screenshots If applicable, add screenshots to help explain your problem.

Desktop (please complete the following information):

  *   Operating System -- macOS Big Sur Version 11.6
  *   Terminal -- tried both iterm2 and alacritty
  *   Version of Neovim -- NVIM v0.5.1 (Installed via Brew)

Additional context I ran PackerStatus and saw this output:
[Image removed by sender. Screen Shot 2021-10-02 at 11 51 49 PM]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F35014096%2F135741834-88595f91-a9b5-4849-9b51-830de80e06e3.png&data=04%7C01%7C%7C6e23c675929d49a5411008d986f23f75%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637689197272900491%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ODp8G4OP5M%2FEe5GEyoOzwqRITf1PLpE5w7ixmwrpTNw%3D&reserved=0>

Basically a ton of plugins that are listed as not loaded. I've tried running PackerInstall but nothing executes...feels like maybe something is broken with Packer for mac users?

@Callendor<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCallendor&data=04%7C01%7C%7C6e23c675929d49a5411008d986f23f75%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637689197272910482%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Y2l5YiA82QUJR%2FxfZ%2B2KGXSXY9h5SfODXhZprCVo%2FSg%3D&reserved=0> is the issue solved after setting nerd font in your terminal?

If it isn't I recommend you to try kitty terminal https://github.com/kovidgoyal/kitty<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkovidgoyal%2Fkitty&data=04%7C01%7C%7C6e23c675929d49a5411008d986f23f75%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637689197272910482%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HTwsyOaomAwMuUIdIsXpuk2qAsixnR67xOzOKRqUiA8%3D&reserved=0>. Kitty terminal supports rendering icons and stuff.
